lnrpc/rpc.proto

```diff
@@ -244,3 +247,29 @@ message ShowRoutingTableRequest {
 message ShowRoutingTableResponse {
     repeated RoutingTableLink channels = 1;
 }
+
+message MultihopPaymentRequest {
+    repeated string LightningIDs = 1;
+    int64 amount = 2;
+}
+
+message MultihopPaymentResponse {
+}
+
+message FindPathRequest {
+    string targetID = 1;
+    int32 numberOfPaths = 2;
+    bool validate = 3;
+    int64 amount = 4;
+    int32 timeout = 5;
+}
+
+message FindPathResponse {
+    message Path {
+        repeated string path = 1;
```

Comment:
The field should be rename to something like: "node_pubkeys". Also perhaps they should just be returned as the raw compressed pubkey bytes of each node within the path.